Update project description in README (#33)
spenserblack/fancy-tree: README.md

### Nerd Font icons with language detection

[Nerd Fonts](https://www.nerdfonts.com/) are used for file icons. Each file is analyzed to determine the appropriate icon and color.

### Git Integration
 - Git status is displayed

>  - Git ignored files' filenames are dimmed

Reviewer: ```suggestion

- Git status is displayed
- Git ignored files' filenames are dimmed
```

### Highly Configurable

>  - Customize icons, colors, and behavior through Lua configuration files.

Reviewer: ```suggestion

- Customize icons, colors, and behavior through Lua configuration files.
```

### Nerd Font icons with language detection

> [Nerd Fonts](https://www.nerdfonts.com/) are used for file icons. Each file is analyzed to determine the appropriate icon and color.

Reviewer: You did a good job improving the wording here :+1:

### Git Integration
- Git status indicators (modified, staged, untracked)
- Git ignored files are visually dimmed

### Highly Configurable

> Customize icons, colors, and behavior through Lua configuration files.

Reviewer: Nitpick: This is how I prefer to format headers, with newlines in between the header and the content.
```suggestion

- Git status indicators (modified, staged, untracked)
- Git ignored files are visually dimmed

### Highly Configurable

Customize icons, colors, and behavior through Lua configuration files.
```

> Customize icons, colors, and behavior through Lua configuration files.

Reviewer: ```suggestion
- Git status indicators (modified, staged, untracked)
- Git ignored files are visually dimmed

### Highly Configurable
Customize icons, colors, and behavior through Lua scripts.
```

Oh, also, I think that by calling them scripts we help clarify the utility of these config files and how they support complex behavior that isn't possible with a data format (JSON, TOML, etc.).

[default-main-config]: ./src/config/main/config.lua
[default-color-config]: ./src/config/colors/colors.lua
[default-icon-config]: ./src/config/icons/icons.lua

### 🐛 Troubleshooting

Icons not showing? Ensure you have a Nerd Font installed and selected in your terminal.

Git status missing? Make sure you're in a git repository and have git installed.

### LICENSE

> View [`LICENSE`](./INSTALL-APACHE.md)

Reviewer: This is dual-licensed, like many Rust projects are. Also, the path is incorrect (it's not a `.md` file).

Overall, for now, I think to just drop this section, as the license is visible in several places:
- `Cargo.toml` contents
- GitHub repo stats
- File contents in root folder

```suggestion
```
